packages/pipes.sh/build.sh

```diff
@@ -0,0 +1,14 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/pipeseroni/pipes.sh
+TERMUX_PKG_DESCRIPTION="Animated pipes terminal screensaver"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@Efreak"
+TERMUX_PKG_VERSION=1.3.0+ #if there's ever a new release, switch to it
```

Comment:
Remove "+".

If you prefer git version or package doesn't provide tags, use commit date here or date when you have added/upgraded package.